superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -87,6 +122,16 @@ export default class ResultSet extends React.PureComponent<
       searchText: '',
       showExploreResultsButton: false,
       data: [],
+      showSaveDatasetModal: false,
+      newSaveDatasetName: `${props.query.tab} ${moment().format(
+        'MM/DD/YYYY HH:mm:ss',
+      )}`,
+      userDatasetsOwned: [],
+      saveDatasetRadioBtnState: DatasetRadioState.SAVE_NEW,
```

Comment:
This is getting too specific. I would just call it "shouldSaveAsNewDataset" or something aline. Imagine what if a future design changes the radio button to a dropdown select?